src/Microsoft.TestPlatform.ObjectModel/PublicAPI/net7.0/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,4 @@
+﻿#nullable enable
```

Comment:
Same question as above, is this just a copy of net6.0 file?

Reply:
no net6 here.